packages/matter.js/src/MatterServer.ts

```diff
@@ -224,7 +224,12 @@ export class MatterServer {
         }
         this.started = true;
         for (const node of this.nodes) {
-            await this.prepareNode(node);
+            try {
+                await this.prepareNode(node);
+            } catch (error) {
+                // TODO: Find a better way how to report back such issues and which nodes errored
```

Comment:
Is there any identifier we could include in error message from the node object that is in scope?

Could also maybe add a todo for retry unless that happens at a higher level.

Reply:
We might have the unique storage key or at least the index. Will add. Good idea

For connect -now that you state it - it should normally not error out but go into "wait" state. Need to check that

Reply:
@lauckhart See additional commit. I now also added uniqueness check for storagekey (which indeed was missing) ... and so we also have that to log. Additoonally: With current code exceptions should only occure on invalid config, so I would not do retry. If a node is commussioned then it should end in "waiting for device to come online for connection" and not throw an error

Reply:
Looks good